tests/jballerina-unit-test/src/test/resources/test-src/types/tuples/tuple_destructure_test.bal

```diff
@@ -109,3 +109,23 @@ function tupleDestructureTest9() returns [boolean, string, [int, float]] {
 
     return [a, b, c];
 }
+
+function tupleDestructureTest10() {
+    Employee[2] employees = [employee1, employee2];
+
+    Employee e1;
+    Employee e2;
+    [e1, e2] = [...employees];
+
+    Employee e3;
+    Employee e4;
+    [e3, e4] = [...[employee1, employee2]];
+
+    Employee e5;
+    Employee e6;
+    [[e5, e6]] = [...[...[employees]]];
+
+    Employee e7;
+    Employee e8;
+    [[e7, e8]] = [[...[...[employee1, employee2]]]];
```

Comment:
Why aren't we asserting anything?

Reply:
Added assertion in the java side

Reply:
Why are we doing it on the Java side? The practice is to assert in b7a.

Reply:
We can do this for just the new code being added.

Reply:
I thought of aligning with the other tests. I'll add it.

Reply:
Fixed